winter/schema/generation.py

```diff
@@ -27,8 +27,8 @@ def generate_swagger_for_operation(view_func, controller, controller_method: Con
     if output_serializer:
         responses[response_status] = output_serializer.class_(**output_serializer.kwargs)
     else:
-        responses[response_status] = openapi.Response(description='Success')
-    # TODO support response dataclasses, preferably implement in dataclasses.py to keep extensibility
+        type_info = get_basic_type_info_from_hint(controller_method.return_value_class)
+        responses[response_status] = openapi.Schema(**type_info.as_dict())
```

Comment:
Maybe move `openapi.Schema` construction to `YASGTypeInfo`, for example `type_info.get_openapi_schema()`.
Because currently, it's not obvious that `YASGTypeInfo` structure is directly mapped to `openapi.Schema` parameters.